src/server/frontend_wayland/xdg_decoration_unstable_v1.cpp

```diff
@@ -118,12 +135,6 @@ void mir::frontend::XdgToplevelDecorationV1::update_mode(uint32_t new_mode)
     send_configure_event(mode);
 }
 
-void mir::frontend::XdgToplevelDecorationV1::set_mode(uint32_t mode)
-{
-    update_mode(mode);
-}
+void mir::frontend::XdgToplevelDecorationV1::set_mode(uint32_t mode) { update_mode(mode); }
```

Comment:
I prefer the four line version as it makes it obvious this is a definition when scanning the file. (However I don't think the guide disallows this, so not a blocker)